pkg/kubelet/dockertools/docker_test.go

```diff
@@ -734,6 +739,14 @@ func TestMakePortsAndBindings(t *testing.T) {
 			if value[0].HostIP != "" {
 				t.Errorf("Unexpected host IP: %s", value[0].HostIP)
 			}
+		case "446":
```

Comment:
I don't think you are actually checking the one-to-many mappings here, as this loop always looks at the first binding `value[0]`...